### PR TITLE
Update storage considerations documentation

### PR DESCRIPTION
**Pull request by SuperQ**
* Improve clarity of internal storage durability.
* Add notes about remote storage.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: docs/storage.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ The directory structure of a Prometheus server's data directory will look someth
 ```
 
 
-Note that a limitation of the local storage is that it is not clustered or replicated. Thus, it is not arbitrarily scalable or durable in the face of disk or node outages and should thus be treated as more of an ephemeral sliding window of recent data. However, if your durability requirements are not strict, you may still succeed in storing up to years of data in the local storage.
+Note that a limitation of the local storage is that it is not clustered or replicated. Thus, it is not arbitrarily scalable or durable in the face of disk or node outages and should be treated as you would any other kind of single node database. Using RAID for disk availiablity, [snapshots](https://prometheus.io/docs/prometheus/latest/querying/api/#snapshot) for backups, capacity planning, etc, is recommended for improved durability. With proper storage durability and planning storing years of data in the local storage is possible.
```
Comment (brian-brazil) [inline docs/storage.md]: etc.

Comment (SuperQ) [reply docs/storage.md]: I'm not sure what you're suggesting.

Comment (brian-brazil) [reply docs/storage.md]: You're missing the period after etc

Comment (SuperQ) [reply docs/storage.md]: No, it's part of the complete sentence of `Using X,Y,Z, etc, is recommended...`.

Comment (SuperQ) [reply docs/storage.md]: How about `Using X,Y, and Z are recommended...`?

**Pull request merged**